dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,8 +407,27 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+
+    // check if application.conf is present in service
+    def isValidProject(p: ProjectRef) = {
+
+      val prj = Project.getProject(p, buildStructure.value)
+
+      val r: PathFinder = {
+        (new File(prj.get.base.getPath) / "src") ** "application.conf"
```

Comment:
Hi @dotta 

In your advice you speak about resourcesDirectories. 
My question is about do you mean the SBT Setting or just the directories (paths). 
I looked at how get the resourcesDirectories' setting on projects (returned by microservicesProjects's task). I didn't find a way to do that. So I think you mean about directories (that I have to hard coded )
I don't if I am clear in my explanation :-/

After this point, I think I will finish quickly the fix.

Many thanks

Fabrice